[RM Ch10] Update Intro (#2010)
cntt-n/CNTT: doc/ref_model/chapters/chapter10.md

<a name="10.1"></a>
## 10.1 Introduction 

> This chapter is dedicated to identifying the challenges and gaps found in the course of the development--to ensure the reference model is adding the strategic and tactical value intended over time. Should a challenge or gap not be identified and it is not already addressed in the model itself, the community may assume it will remain an unknown and, therefore, the community is welcomed to engage with and raise an issue with the appropriate working group(s) to close the gap. In this manner, the Reference Model can continuously improve.

Reviewer: ```suggestion
This chapter is dedicated to identifying the challenges and gaps found in the course of the development of the reference model to ensure that it continues to be of strategic and tactical value intended over time. Should a challenge or gap not be identified that is not already addressed in the model itself, the community may assume it will remain an unknown and, therefore, the community is encouraged to engage with and raise an issue with the appropriate working group(s) to close the gap. In this manner, the Reference Model can continuously improve.
```

<a name="10.2"></a>
## 10.2 Challenges 

> The current challenge is finalizing a stable version from which all stakeholders in the application value-chain can begin to extract the intended value of a Common Cloud Infrastructure. The next maturity level is reached when the Reference Model has stabilized enough for stakeholders to adopt the model into their own application development and deployment cycles.

Reviewer: As there will be multiple RM versions, this is likely to be continuing challenge. Each release will deliver incremental value  -- covered by "intended value".
```suggestion
The continuous challenge is finalizing a stable version from which all stakeholders in the application value-chain can derive the intended value of a Common Cloud Infrastructure. This maturity level is reached when the released Reference Model version is adopted by stakeholders into their application development and deployment cycles.
```
